logging/build.gradle

```diff
@@ -0,0 +1,26 @@
+plugins {
+    id 'jacoco'
+    id 'com.android.library'
+    id 'org.jetbrains.kotlin.multiplatform'
+    id 'maven-publish'
+}
+
+apply from: "../gradle/component.gradle"
+
+group = "com.splendo.mpp"
```

Comment:
See other comment.. I guess now we define this in two places. Still, we can resolve this later.